acceptance/suites/tests/020-external-ca/020_retrieve_new_cert.rb

```diff
@@ -0,0 +1,21 @@
+test_name 'QA-1393 - C62572 - Register Agent with Puppet Master using an External CA'
+
+lockfile = '/opt/puppetlabs/puppet/cache/state/agent_catalog_run.lock'
+lockcopy = '/tmp/foo'
```

Comment:
Better to use `mkstemp` or something similar versus hard-coded temp directories.

Is `lockcopy` actually used?  I only see it in the teardown.